src/main/java/ita/softserve/course_evaluation/configuration/SecurityConfig.java

```diff
@@ -0,0 +1,52 @@
+package ita.softserve.course_evaluation.configuration;
+
+import ita.softserve.course_evaluation.security.jwt.JwtConfigurer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+
+@Configuration
+@EnableWebSecurity
+@EnableGlobalMethodSecurity(prePostEnabled = true)
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+	
+	private final JwtConfigurer jwtConfigurer;
+	
+	public SecurityConfig(JwtConfigurer jwtConfigurer) {
+		this.jwtConfigurer = jwtConfigurer;
+	}
+	
+	@Override
+	protected void configure(HttpSecurity http) throws Exception {
+		http
+				.csrf().disable()
+				.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+				.and()
+				.authorizeRequests()
+				.antMatchers("/").permitAll()
+				.antMatchers("/api/v1/auth/*").permitAll()
```

Comment:
Due to best practice, it's better to directly specify the endpoints you want to have unauthorized access to